cmd/kaidump/kaidump.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/openware/kaigara/pkg/config"
+	"github.com/openware/kaigara/pkg/vault"
+
+	"github.com/openware/pkg/ika"
+	"strings" // Needed to use Split
+	"gopkg.in/yaml.v3"
+)
+
+var cnf = &config.KaigaraConfig{}
+
+func initConfig() {
+	err := ika.ReadConfig("", cnf)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func getVaultService(appName string) *vault.Service {
+	return vault.NewService(cnf.VaultAddr, cnf.VaultToken, cnf.DeploymentID)
+}
+
+func main() {
+	// Parse flags
+	filepath := flag.String("a", "outputs.yaml", "Outputs file path to save secrets from vault")
+	flag.Parse()
+
+	// Initialize and write to Vault stores for every component
+	initConfig()
+	secretStore := getVaultService("global")
+	apps, err := secretStore.ListAppNames()
+	if err != nil {
+		panic(err)
+	}
+	scopes := os.Getenv("KAIGARA_SCOPES")
+	if scopes == "" {
+		panic("Undefined KAIGARA_SCOPES env")
+	}
```

Comment:
You should already have Kaigara scopes and deployment ID since you're using the `config` module
https://github.com/openware/kaigara/blob/0b430d8c5be5311f22da7caf084e7929dfaddd41/pkg/config/config.go#L19-L20

Reply:
Modified to get KAIGARA_SCOPES from config